### DISCUSSION TITLE
Using Sage with service on same system

### DISCUSSION THREAD

**Discussion by xjim1** [OpenSageTV/Community#160] [General]
I've been seeing weird behavior lately, such as 'set watched' and recording padding not working.  I converted my system to running as a service, but when I start Sage on that system, I have always just ran sagetv.exe.  Is that ok or should I install and use the client?

I notice there are 4 sagetv processes running, only 2 when I exit out of the interface.
Thanks for the help.

**Comment by Ugadata** [comment]
Are the service and application running with the same Username?

Did you specify a username for the Sage Service.  By default the service will run with a default service account, which is not the signed on user account.

**Reply by xjim1** [reply]
I checked the service app and it said 'local system account'.  And of course I start the UI under my account.

**Comment by xjim1** [comment]
I tried from another system running the client and same result, so i guess I have two questions:
Should I be using the client on the same system which is running the service?
'Delete and Set Watched' will delete program, but won't flag it as watched if it isn't already flagged as watched.  
  I don't know if this problem started after configuring to run as service or not.  I do remember having to reset a bunch of settings afterwards, but don't config item related to it.

**Comment by Ugadata** [comment]
No need for a separate client on the server.
I see the same issue with 'Delete and Set Watched' as you have.  I run a full client on a separate PC.

Originally, I ran Sage on a single system (client/sever - same machine).  I haven't done it that way for a very long time, except for troubleshooting and verifying things work.

**Comment by xjim1** [comment]
Ok thanks for verifying.
